### PR TITLE
Add Capitec as a sponsor

### PR DESCRIPTION
**Pull request by calvin-brizzi**
They have a bunch of padding on their images already, so I didn't change it,  running the convert command made it too small.

### CODE REVIEW

File: _data/sponsors.yml [hunk 1]
```diff
@@ -85,6 +85,29 @@ sponsors:
...
+      tap to pay-enabled Global One card, we put the value back into our clients’ pockets – helping them to
+      simplify banking and live better.
+      </p>
+      <p>
+      This commitment, combined with our pillars of simplicity, affordability, accessibility and personalised
+      experience, has not only earned us the title of South Africa’s leading retail bank, but also recognition
+      as the best bank in the world.
+      </p>
+      <p>
+      Visit <a href='https://www.capitecbank.co.za/'>https://www.capitecbank.co.za/</a> for more info.
```
Comment (MarijeP) [inline _data/sponsors.yml]: Usually I make it <a href='https://www.capitecbank.co.za/'>www.capitecbank.co.za</a> just for consistency with the others.

Comment (calvin-brizzi) [inline _data/sponsors.yml]:
```suggestion
      Visit <a href='https://www.capitecbank.co.za/'>www.capitecbank.co.za/</a> for more info.
```

**Comment by MarijeP** [comment]
Just the one tiny change, otherwise :+1:

File: _data/sponsors.yml [hunk 2]
```diff
@@ -85,6 +85,29 @@ sponsors:
...
+      tap to pay-enabled Global One card, we put the value back into our clients’ pockets – helping them to
+      simplify banking and live better.
+      </p>
+      <p>
+      This commitment, combined with our pillars of simplicity, affordability, accessibility and personalised
+      experience, has not only earned us the title of South Africa’s leading retail bank, but also recognition
+      as the best bank in the world.
+      </p>
+      <p>
+      Visit <a href='https://www.capitecbank.co.za/'>www.capitecbank.co.za/</a> for more info.
```
Comment (calvin-brizzi) [inline _data/sponsors.yml]:
```suggestion
      Visit <a href='https://www.capitecbank.co.za/'>www.capitecbank.co.za</a> for more info.
```

**Pull request merged**